Specification/src/Ardalis.Specification/ISingleResultSpecification.cs

```diff
@@ -1,9 +1,8 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Text;
-
-namespace Ardalis.Specification
+﻿namespace Ardalis.Specification
 {
+    /// <summary>
+    /// A marker interface for specifications that are meant to return a single entity
```

Comment:
```suggestion
    /// A marker interface for specifications that are meant to return a single entity. Used to constrain methods
    /// that accept a Specification and return a single result rather than a collection of results
```